### PR TITLE
Project 2: Query and update animals

### PR DESCRIPTION
**Pull request by yaseerokino**
### Project requirements
- Add a column `species`.
- Insert the following additional data
-  Commit and rollback transaction update
- Write queries to answer stated questions

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
 SELECT * FROM animals WHERE NOT NEUTERED;
 SELECT * FROM animals WHERE NAME != 'Gabumon';
-SELECT * FROM animals WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3; 
--- SELECT * FROM animals WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3 is quivalent to SELECT * FROM animals WHERE WEIGHT_KG  BETWEEN  10.4 AND 17.3
\ No newline at end of file
+SELECT * FROM animals WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3;
+BEGIN;
+UPDATE animals SET SPECIES='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
```
Comment (Davidosky007) [inline queries.sql:12]: * I suggest you properly state these insertions `Inside a transaction update the animal's table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.`  
* You only started the transaction, made the change (` UPDATE animals SET species = 'unspecified';`) but you didn't verify that change was made and undone for example:  ` SELECT species from animals;` 
  ` ROLLBACK;`
Then
  ` SELECT species from animals; `
`

File: queries.sql [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+BEGIN;
+UPDATE animals SET SPECIES='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET SPECIES='pokemon' WHERE SPECIES IS NULL;
+COMMIT;
```
Comment (Davidosky007) [inline queries.sql:16]: * Kindly Verify that change was made and persists after commit. as stated in the project requirement [Check here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.) 
* check comment above  ⬆️ for HINT 💭 on how to

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @yaseerokino 👋, 

Good job so far! 💪 
* Well done using the right SQL queries ✔️ 
* Descriptive commit messages ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @yaseerokino , 

Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**